transposon/density.py

```diff
@@ -343,23 +348,13 @@ def progress():
     logger.info("Importing transposons, this may take a moment...")
     TE_Data = import_transposons(args.tes_input_file, te_annot_renamer)
 
-    # NOTE Config parser section
-    logger.info("Setting config file...")
-    config = ConfigParser()
-    # NOTE Set this as needed, may have to move elsewhere so people can edit
-    # MICHAEL for the sake of testing, I will leave the options as 100, 100,
-    # 1000, but during our production runs I intend for it to be 500, 500,
-    # 10000
-    config['density_parameters'] = {'first_window_size': 100,
-                                    'window_delta': 100,
-                                    'last_window_size': 1000}
-    with open('config.ini', 'w') as configfile:
-        config.write(configfile)
-
     logger.info("Reading config file...")
+    # NOTE
+    # Michael we could always pass the parser object to the process() function
+    # however that sections seems likely to change, let me know what you think,
+    # if the below variable assignments ought to be moved
```

Comment:
rather than passing the `ConfigParser` instance add the required variables to a dictionary and pass that

that way all the configuration data is in one spot and the parameters are already parsed from the file